### PR TITLE
Feature/front end initial

### PR DESCRIPTION
**Pull request by reeved**
# Description
This PR begins work on #23 and #21 by creating the Top Bar, Game Environment and Bottom Bar components needed to display the Mafia Game.

Please include a summary of the changes here.
- Adds a Top Bar
- Initial GameEnv and BottomBar components created to allow work on other issues

# Demo
(Please include screenshots/GIF here if applicable)

After: 
image

# Type of change:
* New feature (Non-breaking change that adds functionality)

# Checklist:
* Have you merged main into your branch?
* Have you tested your changes to ensure it works as expected and does not break existing functionality?
* If applicable, please ensure sufficient tests are added that is related to the changes.
* Please ensure this PR has a `label`, is linked to an `issue` and is related to a `project`
* If new documentation is required for this change, have you created a new documentation issue that describes the documentation needed?

### CODE REVIEW

**Comment by reeved** [comment]
PR now builds successfully :)

**Comment by ateagit** [comment]
Is this meant to be closed after #30 or are we merging this first and updating later?

**Comment by reeved** [comment]
It will be better to merge this now and update later so that people can start using the container elements imo

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+    <div className='App'>
+      <header className='App-header'>
+        <div style={{ width: '80%' }}>
+          <TopBar userName='Reeve' role='Civilian' />
```
Comment (ateagit) [inline frontend/src/App.js:11]: Will the game screen components be extracted into a game screen page component later?

**Review by ateagit — APPROVED** [review]
LGTM

**Pull request merged**